app/assets/stylesheets/pages.scss

```diff
@@ -0,0 +1,51 @@
+// Place all the styles related to the Pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+
+
+/* General */
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+body {
+  //background-image: url('http://s3.amazonaws.com/codecademy-content/courses/learn-rails/img/bg-static.jpg');
+  background-image: url('https://scontent-dub4-1.xx.fbcdn.net/v/t1.0-9/67945814_2151005818530695_8137754632576303104_n.jpg?_nc_cat=108&_nc_ohc=M_A70dGsqAsAX8p1GmI&_nc_ht=scontent-dub4-1.xx&oh=cbe6bb4bcd2482543c7c8985c357f61c&oe=5EFED1AE');
+  background-repeat: no-repeat;
+  background-size: cover;
+  font-family: 'Kosugi', sans-serif;
+}
+
+.container {
+  margin: 0 auto;
+  width: 960px;
+}
+
+/* Main */
+.main {
+  color: #000;
+  position: relative;
+  top: 100px;
+  text-align: center;
+}
+
+.main h1 {
```

Comment:
As we're using Sass here it might be good to look into how it works and how it is different to CSS. I'd particularly look at nesting in this [documentation](https://sass-lang.com/guide)

Reply:
Thanks for the direction, I'll dive more into utilising Sass.